modulemd/include/modulemd-2.0/modulemd-component-rpm.h

```diff
@@ -200,4 +200,52 @@ modulemd_component_rpm_set_repository (ModulemdComponentRpm *self,
 const gchar *
 modulemd_component_rpm_get_repository (ModulemdComponentRpm *self);
 
+
+/**
+ * modulemd_component_rpm_set_buildroot:
+ * @self: This #ModulemdComponentRpm object.
+ * @buildroot: (in): The #buildroot flag to set for @self.
```

Comment:
Can you explain its purpose here or provide a link to the spec?

Reply:
I fixed the gtk-doc object property linking so clicking on "buildroot" now goes to the right place--which explains the purpose. Is that OK, or do you want me to duplicate it here as well?

Reply:
That works for me.